pandas/_libs/groupby.pyx

```diff
@@ -310,5 +310,88 @@ def group_fillna_indexer(ndarray[int64_t] out, ndarray[int64_t] labels,
                 filled_vals = 0
 
 
+@cython.boundscheck(False)
+@cython.wraparound(False)
+def group_any(ndarray[uint8_t] out,
```

Comment:
If we wanted to we could probably combine `group_any` and `group_all` into one function and just use an argument to tweak the execution.

Reply:
yeah I think making group_any_all is better